src/util/message.cpp

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) 2009-2010 Satoshi Nakamoto
+// Copyright (c) 2009-2019 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+#include <hash.h>            // For CHashWriter
+#include <key.h>             // For CKey
+#include <key_io.h>          // For DecodeDestination()
+#include <pubkey.h>          // For CPubKey
+#include <script/standard.h> // For CTxDestination, IsValidDestination(), PKHash
+#include <serialize.h>       // For SER_GETHASH
+#include <util/message.h>
+#include <util/strencodings.h> // For DecodeBase64()
+
+// We moved a call to boost::get() here from another file. It is a good practice
+// to include the header that provides it (<boost/variant/get.hpp>). However
+// test/lint/lint-includes.sh:90 complains that a new Boost dependency is
+// introduced because even though boost::get() is used in many other places in
+// the code, nobody has bothered to include <boost/variant/get.hpp>.
+//#include <boost/variant/get.hpp> // For boost::get()
```

Comment:
For some reason it is possible to use `boost::get()` without including the header that provides it. Probably included from somewhere else, maybe from QT stuff.

Just drop lines 15-20 or somehow silence the lint script to allow this header to be included (a new boost dependency is not added by this patch)?

Reply:
I dropped lines 15-20